performance_regression_test.py

```diff
@@ -99,7 +105,35 @@ def get_test_xml(self, result):
 
         return test_content
 
-    def display_results(self, results):
+    def generate_stats_json(self, results, cmds=[]):
+        metrics = {}
+
+        for p in self.params.iteritems():
+            metrics[p[1]] = p[2]
+
+        # we use cmds
+        del metrics['stress_cmd']
+        for cmd in cmds:
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+
+        metrics['test_name'] = self.params.id.name
+        metrics['stats'] = results
```

Comment:
I understand, but I think we should have them all stored in ES.
Later when we'll want to compare between runs, we will have all results (nodes + aggregated) in the same place.
I know Shlomi also wants to store whatever we can.